x-pack/plugins/spaces/server/lib/toggle_ui_capabilities.ts

```diff
@@ -5,30 +5,41 @@
  */
 
 import { UICapabilities } from 'ui/capabilities';
+import { Feature } from 'x-pack/plugins/xpack_main/types';
 import { Space } from '../../common/model/space';
 
-export async function toggleUiCapabilities(uiCapabilities: UICapabilities, activeSpace: Space) {
-  toggleDisabledFeatures(uiCapabilities, activeSpace);
+export function toggleUiCapabilities(
```

Comment:
nit: can we change this to `toggleUICapabilities` to be "consistent" with elsewhere?

Reply:
we sure can!